### PR TITLE
[NFC] Fix minor typos in Readme

### PR DESCRIPTION
**Pull request by xgupta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ export LLVM_DIR=<installation/dir/of/clang/11>
 export CLANG_TUTOR_DIR=<source/dir/clang/tutor>
 mkdir build
 cd build
-cmake -DCT_LLVM_INSTALL_DIR=$LLVM_DIR $CLANG_TUTOR_DIR/HelloWorld/
+cmake -DCT_LLVM_INSTALL_DIR=$LLVM_DIR -DCMAKE_EXPORT_COMPILE_COMMANDS=ON $CLANG_TUTOR_DIR/HelloWorld/
```
Comment (banach-space) [inline README.md:74]: Hm, this shouldn't be needed - why would we need this here?

**Comment by banach-space** [comment]
Thanks a ton for submitting this! Quite embarrassing how much I missed!

Is `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` really needed there? Personally I prefer to keep the description of `HelloWorld` to the required minimum. But I might be missing something here!

-Andrzej

File: README.md (hunk 1 above)
Comment (xgupta) [reply README.md]: Sorry, It is not needed here.

**Comment by xgupta** [comment]
Thanks @banach-space  for reviewing! please merge PR.

File: README.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ Building & Testing
 You can build **clang-tutor** (and all the provided plugins) as follows:
 ```bash
 cd <build/dir>
-cmake -DCT_LLVM_INSTALL_DIR=<installation/dir/of/clang/11> <source/dir/clang-tutor>
+cmake -DCT_LLVM_INSTALL_DIR=<installation/dir/of/clang/11> -DCMAKE_EXPORT_COMPILE_COMMANDS=ON <source/dir/clang-tutor>
```
Comment (banach-space) [inline README.md:206]: This is also not needed, right?

**Comment by banach-space** [comment]
Hm, IIUC all changes related to `CMAKE_EXPORT_COMPILE_COMMANDS` can be skipped.

Basically, either we require `CMAKE_EXPORT_COMPILE_COMMANDS` (which generates `compile_commands.json`) or all tools should be run with `--` at the end, e.g.:
```
<build_dir>/bin/ct-code-refactor --class-name=Base --new-name=bar --old-name=foo file.cpp --
```
I think that the latter is cleaner as `ct-code-refactor` won't use `ompile_commands.json` anyway.

Sorry for not making it clearer in my earlier comment.

**Comment by banach-space** [comment]
Would you mind squashing? Then I can merge. I don't want to rely on GitHub's UI (and I want to avoid merge commits too). Thanks a bunch!

**Comment by xgupta** [comment]
> Sorry for not making it clearer in my earlier comment.
No problem -:)

**Comment by banach-space** [comment]
Your patience is much appreciated! :)

**Pull request merged**